src/main/java/com/github/imas/rdflint/RdfLint.java

```diff
@@ -278,8 +280,30 @@ void interactiveMode(RdfLintParameters params, String targetDir) throws IOExcept
         try {
           Query query = QueryFactory.create(line);
           QueryExecution qe = QueryExecutionFactory.create(query, m);
-          ResultSet results = qe.execSelect();
-          ResultSetFormatter.out(System.out, results, query);
+
+          switch (query.getQueryType()) {
+            case Query.QueryTypeSelect:
+              ResultSet results = qe.execSelect();
+              ResultSetFormatter.out(System.out, results, query);
+              break;
+            case Query.QueryTypeConstruct:
+              Model construct = qe.execConstruct();
+              RDFDataMgr.write(System.out, construct, RDFFormat.TURTLE_BLOCKS);
+              break;
+            case Query.QueryTypeDescribe:
+              Model describe = qe.execDescribe();
+              RDFDataMgr.write(System.out, describe, RDFFormat.TURTLE_BLOCKS);
+              break;
+            case Query.QueryTypeAsk:
+              boolean bool = qe.execAsk();
+              System.out.println(bool); // NOPMD
+              break;
+            case Query.QueryTypeUnknown:
+              System.out.println("unknown query type."); // NOPMD
+              break;
+            default:
+              break;
```

Comment:
Can you rewrite following?
- Put println("unknow query type") under default case.
- Remove Query.QueryTypeUnknown case.

Users need to know unsupported in any case - QueryTypeUnknown and QueryTypeJson.

Reply:
Fixed.